python_modules/dagster/dagster/_core/definitions/job_base.py

```diff
@@ -24,127 +19,23 @@ def get_definition(self) -> "JobDefinition":
         pass
 
     @abstractmethod
-    def subset_for_execution(
+    def get_subset(
         self,
-        solid_selection: Optional[Sequence[str]] = None,
+        op_selection: Optional[Iterable[str]] = None,
```

Comment:
should we do the `*` trick to enforce kwargs on call sites here? Might be nice

Reply:
good idea